tensorflow_ros/scripts/classifier_metrics

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python
```

Comment:
Make sure all deps can be installed with use of `rosdep`. Also make sure that a person understands what is happening here. Is this tooling btw specific to `tensorflow_ros`?

Reply:
It shouldn't be, should be general to basically any (multi-label) classifier actually. What's a good place to put it?

Also: it heavily depends on matplotlib, numpy and sklearn. The image_recognition_utils are otherwise quite light, so adding i to that isn't that nice either.

Reply:
OK, the plan for this script is: move it to a new package image_recognition_tools. Because that is not part of this PR, delete it on this branch but add it (back again) later in a new branch (feature/classifier_metrics) off this PR's branch